wd_core.au3

```diff
@@ -359,15 +353,16 @@ EndFunc   ;==>_WD_GetSession
 ;                  Failure - 0 and sets @error to $_WD_ERROR_Exception
 ; Author ........: Danp2
 ; Modified ......:
-; Remarks .......: Separate timeouts can be set for "script", "pageLoad", and "implicit"
-; Related .......:
+; Remarks .......: Separate timeouts can be set for "script", "pageLoad", and "implicit".
+; Related .......: _WD_LastHTTPResult
 ; Link ..........: https://www.w3.org/TR/webdriver#get-timeouts
 ;                  https://www.w3.org/TR/webdriver#set-timeouts
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_Timeouts($sSession, $sTimeouts = Default)
 	Local Const $sFuncName = "_WD_Timeouts"
 	Local $sResponse, $sURL
+	$_WD_HTTPRESULT = 0

```

Comment:
Not needed

Reply:
Agree. Take a look on: [_WD_Timeouts](https://github.com/Danp2/au3WebDriver/pull/247/commits/0ae695a4404fa0e02d78a8de05eb94db2e28d909)